tests/unit/cli/reporter.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { createReport } from '../../../src/cli/reporter';
```

Comment:
Any chance to migrate to the usage of aliases/absolute paths?

Reply:
yeah, i think it's time lol

i can add it in this PR